autogen/agentchat/contrib/vectordb/mongodb.py

```diff
@@ -123,7 +123,11 @@ def _wait_for_document(self, collection: Collection, index_name: str, doc: Docum
             if query_result and query_result[0][0]["_id"] == doc["id"]:
                 return
             sleep(_DELAY)
-
+        if query_result and float(query_result[0][1]) == 1.0:
```

Comment:
We may want to have documents with exactly the same content but different metadata. Like comments from different users. I'd suggest log a warning message instead of raising an error. And it's not a TimeoutError if it's an error. WDYT?

Reply:
Adjusted now with a logger.warning, and check for metadata.